etc/di.xml

```diff
@@ -52,6 +52,24 @@
         </arguments>
     </virtualType>
 
+    <virtualType name="AdyenPaymentHppFacade" type="Adyen\Payment\Model\Method\Adapter">
```

Comment:
This virtualtype is already defined in this file

Reply:
Updated in [98d6afb](https://github.com/Adyen/adyen-magento2/commit/98d6afb209ea447ec045bbd705b77bb701f69a08)